src/wallet/wallet.cpp

```diff
@@ -1400,6 +1400,70 @@ std::optional<PaymentSource> CWallet::SelectPaymentSource(const libzcash::Paymen
     return result;
 }
 
+bool CWallet::HaveSpendingKeyForPaymentSource(const PaymentSource& paymentSource) const {
```

Comment:
Split this refactor into a separate commit.

Reply:
Other refactoring made it possible to just delete this method entirely.